content/en/features/resource-aggregate.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Resource aggregate'
+description: 'Resource aggregate overview'
+date: '2021-06-11'
+lastmod: '2021-06-15'
+categories: [features]
+keywords: [features, resource-aggregate]
+menu:
+  docs:
+    parent: features
+    weight: 10
+toc: true
+---
+
+# Resource Aggregate
+
+Resource Aggregate translates commands to events, stores them to database and publishes them to messaging system.
+
+## ResourceChanged event
```

Comment:
The goal is not to document an event but behaviour / feature. 

## Event Deduplication
## ResourceChanged Event
Document rules, when a received command won't generate a new event. What rules have to be fulfilled.
## ResourceLinksPublished event
same here


by the way, what abou DeviceMetadata deduplication?